src/test/java/uk/gov/companieshouse/api/accounts/utility/ApiLoggingTest.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.companieshouse.api.accounts.utility;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static uk.gov.companieshouse.api.accounts.utility.TestEnvironmentSetupHelper.METHOD_TRACE_ENABLED_KEY;
+import static uk.gov.companieshouse.api.accounts.utility.TestEnvironmentSetupHelper.PERFORMANCE_STATS_ENABLED_KEY;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.companieshouse.api.accounts.utility.logging.ApiLogging;
+
+public class ApiLoggingTest {
+
+  /**
+   * Test that the apiLogging.methodTraceEnabled defaults to False
+   */
+  @Test
+  public void methodTraceNotEnabled() {
```

Comment:
Since we are using JUnit5, could we put `@DisplayName` on each test just to give a descriptive name of what it does. This then can replace the JavaDoc description. :)